MdeModulePkg/Bus/Pci/NvmExpressDxe/NvmExpress.c

```diff
@@ -3,6 +3,7 @@
   NVM Express specification.
 
   Copyright (c) 2013 - 2017, Intel Corporation. All rights reserved.<BR>
+  Copyright (c) Microsoft Corporation.

```

Comment:
Could you add the `<BR>` at the end for doxygen, please?

Reply:
Updated in the latest push. Thanks for reviewing.